.babelrc

```diff
@@ -1,28 +1,26 @@
 {
   "plugins": [
-    "syntax-dynamic-import",
-    "transform-runtime",
-    "transform-object-rest-spread",
-    "transform-class-properties",
-    "transform-es2015-destructuring"
+    "@babel/plugin-syntax-dynamic-import",
+    "@babel/plugin-transform-runtime",
+    "@babel/plugin-proposal-class-properties",
+    "@babel/plugin-transform-template-literals",
+    "@babel/plugin-proposal-object-rest-spread",
+    "@babel/plugin-transform-destructuring"
   ],
   "presets": [
-    "es2015",
-    "react",
-    "stage-0"
-  ],
-  "env": {
-    "start": {
-      "presets": [
-        "react-hmre"
-      ]
-    },
-    "test": {
-      "presets": [
-        "es2015",
-        "react",
-        "stage-0"
-      ]
-    }
-  }
-}
\ No newline at end of file
+    [
+      "@babel/preset-env",
+      {
+        "useBuiltIns": "entry",
+        "targets": {
+          "node": "current",
+          "ie": "11",
+          "firefox": "60",
+          "chrome": "69"
+        }
```

Comment:
firefox: 67
chrome: 75
EdgeHTML/edge: 18/44
ie: 11

Reply:
Are you sure @thomasmaas ? We had some issues of clients using Firefox 60 (with Service Discovery)

Reply:
https://access.redhat.com/articles/2798521

which is updated after each release: we support latest.

Reply:
OK